Allow larger document.browse.margin_width.
By: yozohida@gmail.com
---
 src/config/options.inc |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Index: elinks/src/config/options.inc
===================================================================
--- elinks.orig/src/config/options.inc	2017-11-01 02:41:50.450408746 +0100
+++ elinks/src/config/options.inc	2017-11-01 02:41:50.450408746 +0100
@@ -460,7 +460,7 @@
 
 
 	INIT_OPT_INT("document.browse", N_("Horizontal text margin"),
-		"margin_width", 0, 0, 9, 3,
+		"margin_width", 0, 0, 100, 3,
 		N_("Horizontal text margin.")),
 
 	INIT_OPT_BOOL("document.browse", N_("Document meta refresh"),
